web-client/src/components/figma/BlockStyles.tsx

```diff
@@ -70,6 +75,81 @@ const OrDivider = () => {
   );
 };
 
+const SettingsWrapper = styled.div`
+  padding: 10px;
+`;
+
+const TopPanelWrapper = styled.div`
+  width: 100%;
```

Comment:
The name has to be globally unique.  Many things have TopPanels.  Is this component used globally?

Reply:
Updated